portfolio/src/main/webapp/index.html

```diff
@@ -92,22 +91,14 @@ <h4 class="my-3">To Support</h4>
             </span>
             <h4 class="my-3">To Change</h4>
             <p class="text-muted">
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore
-              harum ex magni, dicta impedit.
+              Too often people use lack of information as an excuse, so by consolidating the world's information into
+              one place that can inform and provide actionable things for you to do as a citizen it is possible to do
+              something to help even if you're at home.
```

Comment:
The first part would fall under "to inform" and I think the rest needs to be reworded. Thinking for future use of our website, the world will be in a different state so adding "even if you're at home" may not always apply. We want to inspire change everywhere in any situation.

Reply:
very good point. changing the wording now.